src/user/guards/is-user-author.guard.ts

```diff
@@ -17,14 +17,17 @@ export class IsUserAuthorGuard implements CanActivate {
     const recipeFromDb = await this.recipeService.findRecipeById(
```

Comment:
This is something that we need to talk about.
In most cases, it's not the best idea to fetch resources from the database inside the decorator applied on the endpoint.

Note that this guard is specific only to the recipe resource. It's not very reusable